jinja2/environment.py

```diff
@@ -283,7 +293,8 @@ def __init__(self,
                  cache_size=400,
                  auto_reload=True,
                  bytecode_cache=None,
-                 enable_async=False):
+                 enable_async=False,
+                 cond_expr_undefined=Undefined):
```

Comment:
For more backwards compatibility this could be set to `None` here and use the value of `undefined` in that case.